pkg/cluster/gitops_test.go

```diff
@@ -0,0 +1 @@
+package cluster
```

Comment:
What's this for?

Reply:
oh yeah that was the start of some test I was adding that wasn't supposed to be part of this, will remove